libraries/AP_RAMTRON/AP_RAMTRON.cpp

```diff
@@ -92,6 +95,30 @@ bool AP_RAMTRON::init(void)
         return false;
     }
 
+    if (has_backup()) {
+        ramtron_logdata = new uint8_t[get_size()];
+        ramtron_logdata_len = get_size();
+
+        if (ramtron_logdata != nullptr) {
```

Comment:
We're still going to register an IO process even though it looks like everything has already gone to hell?

Reply:
This integrity tracker is independent to the data we collected above, with integrity tracker we only record data to the second half of the storage and calculate crc. It is unrelated to fram logging.